vizro-core/examples/default/app_dev.py

```diff
@@ -0,0 +1,337 @@
+"""Example to show dashboard configuration."""
+
+import vizro.models as vm
+import vizro.plotly.express as px
+from vizro import Vizro
+from vizro.actions import export_data
+
+df = px.data.gapminder()
+
+
+page_variable = vm.Page(
+    title="Variable Analysis",
+    description="Analyzing population, GDP per capita and life expectancy on country and continent level",
+    components=[
+        vm.Card(
+            text="""
+                ### Overview
+                The world map provides initial insights into the variations of metrics across countries and
+                continents. Click on Play to see the animation and explore the development over time.
+
+                #### Observation
+                A global trend of increasing life expectancy emerges, with some exceptions in specific African
+                countries. Additionally, despite similar population growth rates across continents, the overall
+                global population continues to expand, with India and China leading the way.  Meanwhile, GDP per
+                capita experiences growth in most regions.
+
+            """
+        ),
+        vm.Graph(
```

Comment:
Can you show me the screenshot? Somehow I get re-directed to the PR when I click on the image that you are linking to! @petar-qb - do you see all the graphs?

Reply:
For me it looks like this:
![Screenshot 2024-01-11 at 14 59 47](https://github.com/mckinsey/vizro/assets/90609403/b48c9faa-cdd3-4d66-b5ad-0a0ee30b7013)

It could be that your screen height is lower than mine, in which case the chart is then not visible. This is a general issue of the layout rather than the container though, as our grid-layout will always try to squeeze in all components on one page. So sometimes the charts might be squeezed too much and are then not visible. In these cases the user has to increase the row min height e.g. if you add the code below to the relevant Page model config, can you see the chart then?


`layout=vm.Layout(grid=[[i] for i in range(4)], row_min_height="450px")`

Reply:
Ah weird why it never loads, but you are right, that will be it!